app/dataSources/dbFetch.tsx

```diff
@@ -30,3 +42,27 @@ export async function getContestantData(keyPrefix: string): Promise<IContestantD
 
     return userList
 }
+
+export async function getLeagueConfigurationData(leagueConfigurationKey: string): Promise<ILeagueConfigurationData[]> {
```

Comment:
I would simplify this whole function since you don't need to get the league configuration for all leagues you can just simplify this by removing the `redis.scan` and just doing a `redis.json.get` and return a sinlge `ILeagueConfigurationData`